scripts/inject-dict.json

```
@@ -73,6 +73,17 @@
     "path": "consts.crowdloan.minContribution",
     "filters": ["humanReadableToken"]
   },
+  {
+    "tpl": "min_nominator_intention",
+    "default": 10,
+    "path": "query.staking.minNominatorBond",
+    "filters": ["humanReadableToken"]
+  },
+  {
+    "tpl": "max_nominator_count",
+    "default": 50000,
+    "path": "query.staking.maxNominatorsCount"
+  },
```

```suggestion
  {
    "tpl": "max_nominator_count",
    "default": 50000,
    "path": "query.staking.maxNominatorsCount"
  },
  {
    "tpl": "max_active_nominator_count",
    "default": 22500,
    "path": "consts.electionProviderMultiPhase.maxElectingVoters"
  },
```

https://polkadot.js.org/docs/substrate/constants#maxelectingvoters-u32